docs/use/originating-assets/index.md

```diff
@@ -0,0 +1,82 @@
+---
+id: originating-assets
+order: 10
+title: Originating Assets 
+---
+## High level user flow
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_9e677fa9504dc2cdfae62dd3b6d1fcab.png)
+
+## Mint NFT in Gateway
+### What is the Gateway
+[Centrifuge Gateway](https://gateway.centrifuge.io/) is the GUI of a Centrifuge Node that allows to securely create and exchange customised documents representing real-world as part of the Centrifuge P2P-protocol and mint NFTs based on these documents.
+
+### Register for Gateway [One time set-up]
+As an asset originator you should have received an invitation email to your email addresss. Click on the `Activate` button in the email. Enter the email shared with the Centrifuge team and choose a safe password according to the required criteria. The 2fa code upon registering is not sent at the moment. So after registering, just login again as normal to receive 2fa code per email. 
+
+### Login to Gateway
+* Go to https://gateway.centrifuge.io/
+* Login with your email address and password
+* You should receive a 2fa security code to your email to complete the login 
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_de80606d6d02f11356486a2ef36d0a3c.png =200x)
```

Comment:
@annamehr, I can't see this image in the preview. Is this a Markdown format issue? I see other images shown in the preview… I think it might that the size parameter ` =200px` is not supported...